webapp/components/wpt-results.html

```diff
@@ -173,56 +164,64 @@
       </paper-toast>
     </template>
 
-    <template is="dom-if" if="{{ pathIsATestFile }}">
-      <test-file-results test-runs="[[testRuns]]"
-                         path="[[path]]"
-                         labels="[[labels]]"
-                         products="[[products]]">
-      </test-file-results>
+    <template is="dom-if" if="[[runsNotFound]]">
+      <info-banner type="error">
+        Failed to fetch test runs.
+      </info-banner>
     </template>
 
-    <template is="dom-if" if="{{ !pathIsATestFile }}">
-      <table>
-        <thead>
-          <tr>
-            <th>Path</th>
-            <template is="dom-repeat" items="{{testRuns}}" as="testRun">
-              <!-- Repeats for as many different browser test runs are available -->
-              <th><test-run test-run="[[testRun]]"></test-run></th>
-            </template>
-            <template is="dom-if" if="[[diffShown]]">
-              <th><test-run test-run="[[diffRun]]"></test-run></th>
-            </template>
-          </tr>
-        </thead>
-
-        <tbody>
+    <template is="dom-if" if="[[hasRuns]]">
```

Comment:
This diff seems needlessly hard to follow. Did you just wrap existing markup in `<template is="dom-if" if="[[hasRuns]]">...</template>`, or is there more here?

Reply:
Yep.